run_dir/static/js/pricing_main.js

```diff
@@ -677,7 +677,8 @@ app.component('v-products-table', {
             this.dataTable = $('#pricing_products_table').DataTable({
                 "paging":false,
                 "info":false,
-                "order": [[ 1, "asc" ]]
+                "order": [[ 1, "asc" ]],
+                colReorder: true
```

Comment:
Breaks the addition of new products in Agreements and Cost calculator.

Reply:
I wasn't aware we could add new products, I don't think I have the right permissions 🤔 I get a server error. 
OK, I will have to look into it then 🤷🏻‍♀️

Reply:
We decided to not include the colReorder for pricing, it would require more work than it's worth 🤷🏻‍♀️